python/ml4ir/base/features/feature_fns/label_processor.py

```diff
@@ -0,0 +1,108 @@
+import tensorflow as tf
+
+from ml4ir.base.features.feature_fns.base import BaseFeatureLayerOp
+from ml4ir.base.io.file_io import FileIO
+
+
+class StringMultiLabelProcessor(BaseFeatureLayerOp):
```

Comment:
Nice code! 
- Do you plan to add a test on this? 
- Another suggestion is to improve a bit the Example in the docstring

Reply:
Yes, to both of those. WIP...

Reply:
It was a placeholder for me to go back and add an example.